changelogs/unreleased/3134-facts-endpoint.yml

```diff
@@ -0,0 +1,4 @@
+description: Added endpoint to get facts
+issue-nr: 3132
```

Comment:
The ticket number in this PR says: `3134`?

Reply:
nice catch